mgmt_cli_test.py

```diff
@@ -512,10 +512,12 @@ def test_backup_rate_limit(self):
         mgr_cluster = manager_tool.get_cluster(cluster_name=self.CLUSTER_NAME) \
             or manager_tool.add_cluster(name=self.CLUSTER_NAME, db_cluster=self.db_cluster,
                                         auth_token=self.monitors.mgmt_auth_token)
-        rate_limit_list = [f'{dc}:{randint(1, 10)}' for dc in self.get_all_dcs_names()]
+        rate_limit_list = [f'{dc}:{randint(15, 25)}' for dc in self.get_all_dcs_names()]
         self.log.info('rate limit will be {}'.format(rate_limit_list))
         backup_task = mgr_cluster.create_backup_task(location_list=self.locations, rate_limit_list=rate_limit_list)
-        task_status = backup_task.wait_and_get_final_status()
+        task_status = backup_task.wait_and_get_final_status(timeout=18000)
```

Comment:
it means that we might wait here longer for some of the existing longevity tests

Reply:
Longevity? You mean backup feature?
In any case, no it doesn't, since if it is a smaller scenario the task will get to one of the "final statuses" way before the timeout, so it won't be affected by it

Reply:
yes, manager feature/sanity tests...
yes, it will get to "final statuses" only if it does not get stuck forever, and in that case, we will only figure it out after we timeout it, after 5 hours...